Dockerfile

```diff
@@ -3,6 +3,12 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
 WORKDIR /source
 ENV ASPNETCORE_URLS=http://+:8080
 
+# Install dependencies
+# hadolint ignore=DL3008
+RUN apt-get update && apt-get install -y --no-install-recommends  \
+	libsqlite3-mod-spatialite \
```

Comment:
We don't need `libsqlite3-mod-spatialite` in the Dockerfile any more, just for CI